geomesa-spark/geomesa-spark-sql/pom.xml

```diff
@@ -61,9 +61,14 @@
         </dependency>
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-cqengine_${scala.binary.version}</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-cqengine-datastore_${scala.binary.version}</artifactId>
             <version>${project.version}</version>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
For the record, the solution I'm going for is to pull out the parts of SparkUtils that use GeoCQEngineDataStore into a separate object. That way Leaflet can use functions like sf2row without needing that dependency.

I'm also realizing that the spark-sql and hbase-spark-runtime cant both have cqengine-datastore as scope 'provided'. To handle both this and not updating the CQs, I'm adding an exclusion for antlr in the cqengine-datastore dependency.

Both of these fixes are in my next commit.